src/styles/blocks/star.scss

```diff
@@ -0,0 +1,14 @@
+.star {
+  width: $star-width;
```

Comment:
The variable `$star-width` is used here, but it seems there is no separate SCSS file for variables as required by the task. Please create a separate SCSS file for variables and ensure `$star-width` is defined there.